fastlane/Fastfile

```diff
@@ -288,6 +288,49 @@ private_lane :emerge_upload do |options|
   )
 end
 
+lane :sync_xcodeproj_references do
```

Comment:
probably good to add a couple params here so we can use it for protobuf and for openapi

Reply:
Hmm, but why do we need it? The current script deals
- with all new `.swift` files which are not referenced in `.xcodeproj`
- and with all deleted `.swift` files